Dockerfile

```diff
@@ -40,10 +40,10 @@ RUN make dist
 FROM cebra-base
 
 # install the cebra wheel
-ENV WHEEL=cebra-0.5.0-py3-none-any.whl
+ENV WHEEL=cebra-0.6.0a1-py3-none-any.whl
 WORKDIR /build
 COPY --from=wheel /build/dist/${WHEEL} .
-RUN pip install --no-cache-dir ${WHEEL}'[dev,integrations,datasets]'
+RUN pip install --no-cache-dir ${WHEEL}'[dev,integrations,datasets,xcebra]'
```

Comment:
remove xcebra; should be in main cebra package

Reply:
done